product_unique_serial/tests/test_for_unicity.py

```diff
@@ -27,7 +27,7 @@
 from copy import deepcopy
 
 from openerp import exceptions
-from . import test_common import TestCommon
+from openerp.addons.product_unique_serial.tests.test_common import TestCommon
```

Comment:
the new test_common.py that was split is meant to be used in downstream modules in other projects.